autoload/keepeye.vim

```diff
@@ -7,6 +7,12 @@ function! keepeye#Callback() abort
   let l:finalmsg = repeat(' ', l:colshlen - l:msghlen) . l:msg
 
   let &statusline = l:finalmsg
+
+  if g:keepeye_system_notification
+    if has('unix') && !has('mac')
+      call system('notify-send ' . shellescape('KeepEye') . ' ' . shellescape(g:keepeye_message))
```

Comment:
Is `notify-send` spread enough ? I mean, is it present in almost all Linux distrib ? If yes then it's ok, otherwise we should check if the executable exists

Reply:
Mmm, good question, I think is kind of a _yes_, at least all gnome desktops have this https://developer.gnome.org/notification-spec/ but, looks like a bigger spec http://www.galago-project.org/docs/index.php (libnotify) I going to research a little more and report back

Reply:
The answer is yes! https://wiki.archlinux.org/index.php/Desktop_notifications#Bash that is a good reading. TL;DR: there is a library (libnotify) that has several servers that are installed on desktop environments (including gnome and kde).

Reply:
Thank you to have checked, I can merge then :)